templates/slurm_detail_stats.sh.epp

```diff
@@ -20,7 +20,7 @@ tfile3=$(mktemp /tmp/nodeinfo.XXXXXX)
 tfile4=$(mktemp /tmp/pending.XXXXXX)
 
 ##Dump info about all running jobs into a temp file; get the list of all nodes in the system
-"${slurm_path}"/squeue -t running -O Partition,NodeList:50,tres-alloc:70,username | grep -v TRES_ALLOC | awk '{print $1","$2","$3","$4}' > "${tfile2}"
+"${slurm_path}"/squeue -t running -O Partition:50,NodeList:250,tres-alloc:70,username | grep -v TRES_ALLOC | awk '{print $1","$2","$3","$4}' > "${tfile2}"
```

Comment:
Minor suggestion...

Since the `Partition,NodeList:50,tres-alloc:70,username | grep -v TRES_ALLOC` is used both here on line 23 & later on line 173, would it make sense to set that whole string as a variable? Basically minimize the chance of have the output be different in line 23 versus line 173 ?

Reply:
@billglick I'd like to keep them not as a variable for now if that's ok.  I definitely see the reasoning there and don't think it's necessarily a bad idea.  Would like to keep separate for now.  I have my own side branch to clean up this whole script and I'll bake this change in as part of that effort.